iroha/src/permissions.rs

```diff
@@ -261,7 +261,7 @@ impl PermissionsValidator for AllowAll {
 }
 
 impl From<AllowAll> for PermissionsValidatorBox {
-    fn from(_: AllowAll) -> Self {
+    fn from(AllowAll: AllowAll) -> Self {
```

Comment:
Is this even legal :smile: ? Anyway I think the underscore should be used here, as the type is not used in the body of the function.

Reply:
As legal as `fn func((): ()) {}`. If we will add arguments this code will break :)